src/app/modules/shared/components/tables/mat-fsp-table/mat-fsp-table.component.scss

```diff
@@ -0,0 +1,13 @@
+table {
+    width: 100%;
+}
+.mat-form-field {
+    margin-left: 28px;
```

Comment:
Using `px` for adaptive frontend project is a code smell. Please, change this to relative units, like percents, or use a [bootstrap classes](https://getbootstrap.com/docs/4.3/utilities/spacing/). For font same.